client/payments/data/types.ts

```diff
@@ -0,0 +1,32 @@
+export type PaymentGateway = {
+	id: string;
+	title: string;
+	description: string;
+	order: number | string;
```

Comment:
Could we change this to `number | ""`? As I assume it is only a string when it is not defined - empty string.

Reply:
[test.reducer.ts](https://github.com/woocommerce/woocommerce-admin/pull/6918/files#diff-0b6b2736f76cb2bf1deb0ad24a104fa9ac3510e4439cbee90cf04c45a1c48fc4) started complaining about the type when I changed it to `number| ""`. I left it as `number | string` for now. I'm gonna look into it.